negative utilitarianism

```diff
@@ -0,0 +1,38 @@
+def negativeBentham(agent, cell):
+    cellSiteWealth = cell.sugar + cell.spice
+    globalMaxCombatLoot = cell.environment.maxCombatLoot * 2
+    cellMaxSiteWealth = cell.maxSugar + cell.maxSpice
+    if cell.agent is not None:
+        cellSiteWealth += min(cell.agent.wealth, globalMaxCombatLoot)
+        cellMaxSiteWealth += min(cell.agent.wealth, globalMaxCombatLoot)
+    cellNeighborWealth = cell.findNeighborWealth()
+    globalMaxWealth = cell.environment.globalMaxSugar + cell.environment.globalMaxSpice
+    
+    positiveCellValue = 0
+    negativeCellValue = 0
+    selfishnessFactor = agent.selfishnessFactor
+    
+    for neighbor in agent.neighborhood:        
+        neighborValueOfCell = neighbor.decisionModelFactor * ((certainty * proximity) * ((extent * (intensity + duration)) + (discount * (futureExtent * (futureIntensity + futureDuration)))))
+        
+        if neighbor != agent:
+            if cell != neighbor.cell:
+                if neighborValueOfCell < 0:
+                    negativeCellValue += neighborValueOfCell
+            else:
+                if neighborValueOfCell < 0:
+                    negativeCellValue += neighborValueOfCell
+        else:
+            positiveCellValue += neighborValueOfCell
+        
+        if selfishnessFactor != -1:
+            if neighbor == agent:
+                positiveCellValue *= selfishnessFactor
+            else:
+                if neighborValueOfCell < 0:
+                    negativeCellValue *= (1 - selfishnessFactor)
+
+    if positiveCellValue >= abs(negativeCellValue):
```

Comment:
This function should return _both_ the positive and negative values of the cell. This is to resolve ties where two potential cells have the same negative value (as in, they case the same amount of harm) in which case the cell with the best positive consequences should be preferred.

This comparison of different cells needs to take place outside this function (and within `agent.py`) which is why returning a tuple or list of both is necessary.